.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
   test:
     docker:
-      - image: cimg/node:14.15.4
+      - image: cimg/node:16.14.2
```

Comment:
Also had to update Node version for CircleCI since the newly added @typescript-eslint/eslint-plugin was incompatible with the existing version. Ref: [error](https://app.circleci.com/pipelines/github/Instawork/hyperview/1230/workflows/05b7cc96-3360-4eed-8c0e-040a9551c88f/jobs/1735?invite=true#step-102-14)